scipy/signal/filter_design.py

```diff
@@ -2842,6 +2842,10 @@ def butter(N, Wn, btype='low', analog=False, output='ba', fs=None):
 
     The ``'sos'`` output parameter was added in 0.16.0.
 
+    If you use the transfer function syntax [b, a], you might encounter numerical problems.
```

Comment:
I would suggest a bit more informative version

```
    The conversion between roots and polynomial coefficients is a numerically 
    sensitive operation. Thus, conversion to a Transfer Function can be unstable
    even for N = 4 and above. It is recommended to keep the SOS representation.
```

or something along those lines. 

Also we need to keep an eye on the PEP8 stuff. The line is longer than 80 characters.

Reply:
@ilayn Thank you for your review. I fixed it based on your review. Please check it again.